src/scripts/main.js

```diff
@@ -1,3 +1,102 @@
 'use strict';
 
-// write code here
+const baseURL
+= 'https://mate-academy.github.io/phone-catalogue-static/api/phones.json';
+const idURL
+= 'https://mate-academy.github.io/phone-catalogue-static/api/phones/';
+
+const allPhonesList = getAllPhones();
+
+allPhonesList
+  .then(data => {
+    const phoneIds = data.map(phone => phone.id);
+
+    const allSuccessfulDetails = getAllSuccessfulDetails(phoneIds);
+
+    const firstReceived = getFirstReceived(phoneIds);
+
+    const firstThreeReceived = getFirstThree(phoneIds);
+
+    allSuccessfulDetails
+      .then(responses => {
+        const successfullResponses = responses.filter(response => {
+          return response.status === 'fulfilled';
+        });
+
+        const successfulDetailsList
+        = successfullResponses.map(response => response.value);
+
+        createNotification(successfulDetailsList);
+      });
+
+    firstReceived
+      .then(phone => createNotification([phone], 'First received'));
```

Comment:
```suggestion
      .then(phone => createNotification([phone], 'first_received'));
```
передавай сразу в таком виде, чтобы не делать replace